Remotion/Web/Core/Res/Themes/NovaViso/Image/sprite.svg

```diff
@@ -38,7 +38,14 @@
     <rect x="0.5" y="0.5" height="15" width="15" fill="#ffffff" rx="2" ry="2" stroke="#979797"/>
   </svg>
   <svg class="sprite-part" id="CheckBoxNull" width="16px" height="16px" viewBox="0 0 16 16">
-    <rect x="0.5" y="0.5" height="15" width="15" fill="#979797" fill-opacity="0.5" rx="2" ry="2" stroke="#979797" />
+    <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
```

Comment:
you didn't use the variant from the comments here. intentional or oversight?

Reply:
Oversight, I'll fix it, sorry